packages/caravan-clients/src/__tests__/client.test.ts

```diff
@@ -5,21 +5,32 @@ import {
   ClientBase,
   UTXO,
   BlockchainClientError,
-} from "./client";
-import * as bitcoind from "./bitcoind";
-import * as wallet from "./wallet";
+} from "../client";
+import * as bitcoind from "../bitcoind";
+import * as wallet from "../wallet";
 import BigNumber from "bignumber.js";
 
+import {
```

Comment:
this shouldn't be needed with the globals. You might need to set something up in eslint as well?

Reply:
Yep, I've configured eslint a bit to improve the import orders